graphqlapi/src/main/java/io/stargate/graphql/schema/fetchers/dml/DmlFetcher.java

```diff
@@ -113,31 +98,9 @@ protected DmlFetcher(
           Term keyTerm = toCqlTerm(keyType, entry.get("key"));
           Column.ColumnType valueType = mapType.parameters().get(1);
           Term valueTerm = toCqlTerm(valueType, entry.get("value"));
-          relation = Relation.mapValue(column.name(), keyTerm).isEqualTo(valueTerm);
+          relation = operator.buildRelation(column.name(), keyTerm, valueTerm);
         } else {
-          Term rightTerm = toCqlTerm(column, condition.getValue());
-          switch (condition.getKey()) {
-            case "eq":
-              relation = relationStart.isEqualTo(rightTerm);
-              break;
-            case "notEq":
-              relation = relationStart.isNotEqualTo(rightTerm);
-              break;
-            case "gt":
-              relation = relationStart.isGreaterThan(rightTerm);
-              break;
-            case "gte":
-              relation = relationStart.isGreaterThanOrEqualTo(rightTerm);
-              break;
-            case "lt":
-              relation = relationStart.isLessThan(rightTerm);
-              break;
-            case "lte":
-              relation = relationStart.isLessThanOrEqualTo(rightTerm);
-              break;
-            default:
-              throw new IllegalStateException("Unsupported relation type " + condition.getKey());
-          }
+          relation = operator.buildRelation(column.name(), toCqlTerm(column, condition.getValue()));
```

Comment:
Same remark as `buildCondition`. It would be nice to have all the details in the enum, and treat every operator the same here (the way it's already done for "simple" ones):
```java
      for (Map.Entry<String, Object> condition : clauseEntry.getValue().entrySet()) {
        FilterOperator operator = FilterOperator.fromFieldName(condition.getKey());
        relations.add(operator.buildRelation(column, condition.getValue()));
      }
```